app/src/main/java/org/mifos/mobilebanking/ui/activities/base/BaseActivity.java

```diff
@@ -252,4 +254,12 @@ public int stackCount() {
         return getSupportFragmentManager().getBackStackEntryCount();
     }
 
+
+    public static void hideKeyboard(Activity activity) {
+        InputMethodManager inputMethodManager =
+                (InputMethodManager) activity.getSystemService(
```

Comment:
Please use Context instead of Activity

Reply:
Actually Activity is used for using the getCurrentFocus( ) . What I can do is pass context as the parameter and convert it to activity something like this:
 public static void hideKeyboard(Context context) {
        Activity activity = (Activity) context;
        InputMethodManager inputMethodManager =
                (InputMethodManager) context.getSystemService(
                        Activity.INPUT_METHOD_SERVICE);
        inputMethodManager.hideSoftInputFromWindow(
                activity.getCurrentFocus().getWindowToken(), 0);
    }